docs/adr/0037-use-nx-for-monorepo.md

```diff
@@ -0,0 +1,48 @@
+# Use nx for monorepo tooling
+
+- Status: proposed
+- Deciders: Ben Bangert, Wil Clouser, Dan Schomburg
+- Date: 2023-03-13
+- Obsoletes: 0019-use-workspace-dependencies
+
+## Context and Problem Statement
+
+Our monorepo using a yarn workspace model has a few limitations we've experienced such as slow builds, using bash scripts for running tasks, having to make each new library or component as its own package with its own dependencies, and an inflexible structure for refactoring our shared libraries.
```

Comment:
I think another major sticking point is lack of consistency between packages. Each package is setup in its own way with little standardization between packages. This makes any cross cutting concern difficult to maintain.

Reply:
Added a blurb about the inconsistency on tooling for maintenance.